internal/factsengine/gatherers/saphostctrl_test.go

```diff
@@ -0,0 +1,109 @@
+package gatherers // nolint:dupl
```

Comment:
Shouldn't this be `gatherers_test`? Having troubles with package names usually means that we have to use some sort of dependency injection to inject the mock for some struct

Reply:
Nope. The file belongs to the same package